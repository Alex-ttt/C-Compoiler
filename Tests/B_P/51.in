int func(int **){
	return 0;
}

void main(){
	int a[5], b;
	int *c=&a[1];
	func(&c);
}