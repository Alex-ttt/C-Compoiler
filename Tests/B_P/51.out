 Symbols Table:
   func  int function func(
           pointer to pointer to int
  )
    {
      Statements:
      return
        0
    }
   main  void function main(
  )
    {
      Statements:
      func
      (
        &
          c
      )
      Symbols Table:
              a array[5] of int
              b int
              c pointer to int
    }
